### ISSUE TITLE
RigidBody when moving ignores collision with Character Controller

### ISSUE THREAD

**Issue by PhyresiCompany** [FlaxEngine/FlaxEngine#1237]
**Flax 1.6.6343**

- My `RigidBody` traverses my `CharacterController` when moving.

https://github.com/FlaxEngine/FlaxEngine/assets/101017436/23dc56be-8a9c-43e0-9fe3-cf8778dcbaf7

But if the `RigidBody` is not moving my `Character Controller` can collide with the RigidBody.

https://github.com/FlaxEngine/FlaxEngine/assets/101017436/c6fff0b8-1926-4d1f-ad10-2160295c7590

**My Project:** 
[Test1.zip](https://github.com/FlaxEngine/FlaxEngine/files/11964774/Test1.zip)

**Comment by PhyresiCompany** [comment]
@mafiesto4 Is this a bug in Flax or something I did wrong in my project?

**Comment by mafiesto4** [comment]
You're right, it looks like Character Controller has issues when it comes to other objects colliding with it. I think it might be PhysX problem. We could overcome it by adding custom, hidden capsule collider that would block other objects from intersecting the CC.

**Comment by NoriteSC** [comment]
the 
rb.AddMovement(move); is not working because 
characterController.Move acuity controls the collision
unless rb can have control over CC it will not work
